components/ui/NavItem.js

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link";
+import classNames from "classnames";
+import styles from "./Navbar.module.css";
+
+export const NavItem = ({ title, link, isActive }) => {
+    return (
+        <li
+            className={classNames(styles.navbarLinks, {
+                [styles.isLinkActive]: isActive,
+            })}
+        >
+            <Link href={link}>
+                <b>{title.slice(0, 2)}</b> {title.slice(3)}
```

Comment:
This is tricky, because the title can be changed at any time.
So if we start using 001 and 002 ... this would break immediatly.
What you can do is to add in an extra variable (e.g. `index`) and use that as the bold content